### ISSUE TITLE
2 Realm Portal Rune

### ISSUE THREAD

**Issue by Ermak0** [DestinyNetwork/DungeonRealms-Bug-Report#121]
Sometimes i spawn with two Realm Portal Rune and rarely i spawn without the Realm Portal Rune and the Character Journal.

![2015-09-16_12 42 18](https://cloud.githubusercontent.com/assets/14290685/9902674/aac811fe-5c70-11e5-8fc8-e2d241cbb147.png)

**Comment by Ermak0** [comment]
Today i spawned with two Heartstone

![2015-09-17_13 54 07](https://cloud.githubusercontent.com/assets/14290685/9932372/b19f0572-5d43-11e5-90a8-7b129383533e.png)

**Comment by NoahMCGamer** [comment]
Yeah, same thing happened to me, you can get rid of extra hearthstones by putting them in bank, then they disappear, but it doesn't work for realm portal rune. And some people are saying you have to /friend someone to get character journal. Hope they fix this bug soon.
